frame/session/src/lib.rs

```diff
@@ -140,6 +140,8 @@ use sp_std::{
 pub use pallet::*;
 pub use weights::WeightInfo;
 
+pub const LOG_TARGET: &str = "runtime::session_historical";
```

Comment:
I get a compile error about `LOG_TARGET` not being used in sessions package without it. I think the error is because it's only used in the migrations folder and not in the module where it's defined. I could also move `LOG_TARGET` to the migrations folder and remove pub. I'll remove it and share the error that pops up.

Reply:
Did you try `pub(crate)`?

Reply:
It works with `pub(crate)` but it seems cleaner to just move it. So i've moved it :)